src/FileOnQ.Imaging.Heif/IHeifImage.cs

```diff
@@ -2,9 +2,35 @@
 
 namespace FileOnQ.Imaging.Heif
 {
+	/// <summary>
+	/// A unprocessed Heif image.
+	/// </summary>
 	public interface IHeifImage : IDisposable
 	{
+		/// <summary>
+		/// Gets the embedded thumbnail if exists.
+		/// </summary>
+		/// <returns>
+		/// An instance of <see cref="IImage"/> which
+		/// contains the embedded thumbnail.
+		/// </returns>
+		/// <exception cref="HeifException">
+		/// If an error occurs during retrieving the embedded thumbnail, an exception will be
+		/// thrown which contains an error code explaining the problem.
```

Comment:
Can you use `<see cref="" />` to link to the error code object

Reply:
@ahoefling I THINK the code I have SHOULD work per MS. BUT I can change it if desired: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags#exception

Reply:
Oh 🤦‍♂️ I wasn't clear I what I wanted. The `<exception>` tag is correct and looks good to me. I want you to look at changing line 19 to something like this
```suggestion
		/// thrown which contains a <see cref="HeifException.Error" /> explaining the problem.
```